app/helpers/votes_counter.rb

```diff
@@ -0,0 +1,6 @@
+def tally_votes (object)
+  #counts all of the votes for the given object
+  all_votes = object.votes.to_a
+  all_votes.reduce(0) { |sum, vote| sum + vote.value }
```

Comment:
Kinda the same as above, it's much more OOP-y to do `question.tally_votes` than some globally available `tally_votes(question)`. Since we have votes on a lot of things, this might be a module (`Votable`?) that gets included on everything we have votes on.